tests/Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN sh artifacts \
 
 WORKDIR /src/tests
 # Note: We have to run since fetch doesn't write!
-RUN deno run --lock-write --lock=lock.json pad.ts \
+RUN deno cache --lock-write --lock=lock.json pad.ts \
```

Comment:
cool, this didn't used to work!

Reply:
Yeah, I don't really know when it was "fixed" but yeah now it's ok :)

Reply:
I suppose should delete the comment above :)